combat-trainer.lic

```diff
@@ -1915,11 +1918,16 @@ class AbilityProcess
     @barb_buffs
       .select { |name| Flags["ap-#{name}-expired"] }
       .each do |name|
-        Flags.reset("ap-#{name}-expired")
-        activate_barb_buff(name)
-        echo "Activated barb_buff #{name}!" if $debug_mode_ct
-        waitrt?
-        pause 6 if get_data('spells').barb_abilities[name]['type'].eql? 'meditation'
+        if mana < @barb_buffs_inner_fire_threshold
+          echo "Not activating barb_buff #{name} until have at least #{@barb_buffs_inner_fire_threshold} inner fire" if $debug_mode_ct
```

Comment:
💡 Maybe include in the echo what the current inner fire level is?

Reply:
Yeah, that'd be a nice addition